lib/googleauth/client_id.rb

```diff
@@ -63,13 +64,14 @@ class << self
       #       & secrets in source. See {#from_file} to load from
       #       `client_secrets.json` files.
       def initialize(id, secret)
+        CredentialsLoader.warn_if_cloud_sdk_credentials(id)
```

Comment:
We haven't been style-consistent with this repo in the past, but moving forward I'd like to be more so (so eventually we can set the rubocop file to be similar to the one in google-cloud-ruby). Our ruby style is "seattle style" which, among other things, means minimal use of parens. Here, and in the _new_ code you've added below, please drop parens for most normal method calls. e.g.

```ruby
        CredentialsLoader.warn_if_cloud_sdk_credentials id
```

Reply:
Is there a definitive reference for this I can refer to?

Reply:
I don't think so. Best I can point you to is the `.rubocop.yml` files in google-cloud-ruby. ([Example](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-storage/.rubocop.yml)). The existing code isn't yet compliant with those guidelines, but we should start moving in that direction. Ideally, all our libraries' rubocop settings would be essentially identical.